adminSiteClient/ChartEditorPage.tsx

```diff
@@ -53,7 +58,8 @@ import {
     VisionDeficiencyEntity,
 } from "./VisionDeficiencies.js"
 import { EditorMarimekkoTab } from "./EditorMarimekkoTab.js"
-import { BAKED_BASE_URL } from "../settings/clientSettings.js"
+import { EditorExportTab } from "./EditorExportTab.js"
+import { runDetailsOnDemand } from "../site/detailsOnDemand.js"
```

Comment:
Is it okay to import from the site into the admin client code or have I committed a sin?

Reply:
Afaik, it's allowed. We already do it in `VariableEditPage.tsx`. It just shifts around where our bundler puts things:
- `admin.js` - only used in the admin
- `owid.js` - only used in the site
- `common.js` - used in both

hence bundlewatch complaining that you've increased the size of common.js even though owid.js decreased in size by a proportionate amount.

Reply:
Ahh that makes sense, thanks for explaining!